src/sass/variables/_Color.Variables.scss

```diff
@@ -38,8 +38,9 @@ $ms-color-white: #ffffff !default;
 
 // Shared
 $ms-color-sharedPinkRed10: #750b1c !default;
-$ms-color-sharedRed30: #a4262c !default;
```

Comment:
@mikewheaton just confirmed that removing this will break a scenario @leddie24 is working on. Let's work to address this offline before publishing a new version.